### PR TITLE
Fm/sequents

### LINKED ISSUE #21: Anonymous fm sym
https://github.com/Nolan330/fm/blob/4e91b7cc9c2e349454ebbcd561e8d723490d25ee/src/fm/macros.clj#L5-L10

might muddy the macro a bit, but it's worth being able to give these a name like you can with `fn`:

```clojure
(fm assoc ^{,,,} [x] (assoc m :k x)) ; => #function[the.ns/assoc--28219]
```

### PR DESCRIPTION
**Pull request by usernolan**
They live. 🌀🌀🌀

Much to discuss.

### CODE REVIEW

**Comment by Nolan330** [comment]
Fixes #21

File: src/fm/api.cljc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+(def fm?
+  (comp boolean :fm/sym meta))
+
+(def sequent?
+  (comp boolean :fm/sequent meta))
+
+(def anomaly?
+  (partial s/valid? :fm/anomaly))
+
+(def args-anomaly?
```
Comment (localshred) [inline src/fm/api.cljc:15]: I was just thinking about this this morning, good call.

**Comment by usernolan** [comment]
@localshred thinking this has left `wip` status. let me know if you have any feedback prior to mashing green ✅

**Comment by localshred** [comment]
🙌 🙌 🙌  Review impending... 🙌 🙌 🙌

**Review by localshred — APPROVED** [review]
I'll be the first to admit a lot of this code is over my head (in terms of trying to tie everything together), but I get the general gist of things. You're a mad genius for getting this out the door. Mad props. 😍

**Comment by usernolan** [comment]
i don't love the way the `form-fn`s came out either. it's kind of the definition of spaghetti, but i ran into a bit of difficulty with a lot of the non-pasta models i tried 😂. more to come on that front.

the sequents themselves result in some funky looking code, without a doubt, but they stay pretty close to home in terms of reader/writer expectations ("so he hopes"). all else considered, glad to finally be getting these in.

**Pull request merged**